css/index.css

```diff
@@ -1,3 +1,380 @@
+	align-items: flex-start;
+    flex-flow: row wrap;
+    
+}
+
+.services section{ 
+
+	width: 418px;
+	border: solid 1px green;
+	padding: 20px;
+	margin-bottom: 40px;
+	background-color: lightgrey;
+
+}
+
+.services-header-img{
+	margin: 30px 0 30px;
+}
+
+.container2{
+	width:876px;
+	margin: 0 auto;
+}
+
+.services-facts{
+	display:flex;
+	justify-content: space-between;
+  
+}
+
+.services-facts section{
+	padding-right: 20px;
+	
+}
+
+.services-facts section p{
+	margin-bottom: 20px;
+}
+
+ul {
+	list-style-type: square;
+	margin-left: 20px;
+	line-height: 1.5em;
+
+}
+
+footer{
+	margin: 50px 0 100px;
+	text-align: center;
+}
+
+.topservices{
+	margin-bottom: 40px;
+}
+
+.services section a{
+	 display: inline-block;
+	 border: solid 1px black;
+	 padding: 8px 35px;
+	 margin-top: 20px;
+	 border-radius: 5px;
+	 text-decoration: none;
+     color:inherit;
+     
+}
+
+.main-header{
+    margin-left: 200px;
+}
+
+
+@media (max-width: 800px) {
+
+
+    .content {
+        display: flex;
+        justify-content: space-between;
+        align-content: center;
+        width: 10px;
+        margin-left: auto;
+        margin-right: auto;
+        
+    }
+
+        
+        
+        .navigation a:hover {
+            background-color: green;
+            color: white;
+        }
+        
+        .navigation{
+            display: flex;
+            align-items: center;
+            justify-content: space-between;
+        }
+
+        .content {
+            display: flex;
+            justify-content: space-between;
+            align-content: center;
+            width: 670px;
+        }
+        
+
+        .contacto{
+        margin-left: 100px;
+        margin-top: 20px;
+            width: 200px;
+        
+        }
+
+    .middle-img{
+        width: 650px;
+        margin-left: 100px;
+        
+    }
+    
+    .navigation{
+        
+        font-size: 10px;
+        margin-left:120px;
+        border-radius: 10px;
+            color: grey;
+            font-size: 12px;
+            padding: 18px 15px;
+            text-decoration: none;
+            width: 600px;
+
+    }
+
+    .main-header{
+        margin-left: 190px;
+        margin-bottom: 4px;
+        margin-top: 4px;
+        width: 440px;
+        
+    }
+
+}
+
+
+@media (max-width: 500px){
+     .content{
+         display: flex;
+         flex-direction: column;
+         width: 440px;
+         
+         
+     }
+
+     .contacto{
+        margin-left: 220px;
+      }
+
+    .navigation{
+        
+        display: flex;
+        flex-direction: column;
+        width: 600px;
+        height: 100px;
+        font-size: 200px;
+        margin-bottom: 80px;
+        
+    }
+
+    .navigation a{
+        font-size: 18px;
+        margin-bottom: 15px;
+    }
+
+    .middle-img{
+
+        width: 400px;
+        height: 120px;
+        margin-left: 220px;  
+    }
+
+    .content section{
+       margin-top: 20px;
+    }
+
+
+}
+
+
+
```

Comment:
I'm seeing a lot of hard-coded pixel values and not enough percentage values for widths. Try using percentages, it will stop the scroll-bar problem from happening.